src/models/MySqlModel.ts

```diff
@@ -64,13 +64,17 @@ export class MySqlModel {
     if (val === null || val === undefined) {
       return null;
     }
-
     switch (type) {
       case 'number':
         return Number(val);
       case 'json':
         return JSON.stringify(val);
+      case 'object':
```

Comment:
L72: @briri this is breaking /login because a date is of type 'object' and it is being converted into a string, instead of being handled by the `default` case.

Reply:
Thanks @briri. LGTM.